frontend/src/components/ImplementationReportView/DialectCompliance.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Card, Table } from "react-bootstrap";
+import { ImplementationMetadata } from "../../data/parseReportData";
+
+const DialectCompliance: React.FC<{ specificData: ImplementationMetadata }> = ({
+  specificData,
+}) => {
+  const dialectMapping: { [key: string]: string } = {
+    "draft2020-12": "https://json-schema.org/draft/2020-12",
```

Comment:
This seems like an oddly specific place for this mapping.

Does it not exist in the frontend yet? If not, let's move it somewhere less specific to this page.